docs/agent-install-standalone-windows.md

```diff
@@ -0,0 +1,115 @@
+# Install to Windows using a ZIP file
+
+Install the SignalFx Smart Agent to a Windows host using a standalone package in
+a ZIP file.
+
+## Prerequisites
+
+* Windows Server 2012 or higher
+* Windows PowerShell access
+* Windows decompression application, such as WinZip
```

Comment:
Windows can handle this natively for a long time now.

Reply:
I add clarifying text.